features/org.wso2.carbon.appmgt.publisher.feature/src/main/resources/publisher/apis/v1/tags_api_router.jag

```diff
@@ -65,6 +65,7 @@ routeManager.register('GET', 'publisher', '/publisher/api/tag/{type}/{id}', func
 
     for (var index in tags) {
         tagArray.push({id: counter, name: tags[index]});
+        counter++;
```

Comment:
Seems that `rxtManager.registry.tags(path)` (in line [60](https://github.com/wso2/carbon-appmgt/pull/147/files#diff-53e54f1a4942ce1e3bb013610e2a5b6fL60)) returns a string array (see ['carbon' Jaggery module](https://github.com/wso2/jaggery-extensions/blob/master/carbon/module/scripts/registry/registry-osgi.js#L290)). Therefore this for-each loop should be changed into a for loop. Fixed in commit a37b5a2ea54878ddb107bbbb4f173a9ad5e880e3 of PR #184.